pkg/controllers/workloadrebalancer/workloadrebalancer_controller_test.go

```diff
@@ -462,8 +462,8 @@ func newClusterRoleObjectReference(obj *rbacv1.ClusterRole) appsv1alpha1.ObjectR
 
 // Helper function for generating random suffix
 func randomSuffix() string {
-	max := big.NewInt(10000)
-	n, err := rand.Int(rand.Reader, max)
+	maxInt := big.NewInt(10000)
+	n, err := rand.Int(rand.Reader, maxInt)
```

Comment:
```suggestion
	n, err := rand.Int(rand.Reader, big.NewInt(10000))
```
How about this? Saves a variable.